lib/calendar_assistant/has_duration.rb

```diff
@@ -8,6 +8,11 @@ module ClassMethods
       def duration_in_seconds start_time, end_time
         (end_time.to_datetime - start_time.to_datetime).days.to_i
       end
+
+      def cast_date(date, time_zone = Time.zone.name)
+        return date if date.is_a?(Google::Apis::CalendarV3::EventDateTime)
```

Comment:
I don't think we have test coverage on this; and we aren't yet passing this method a date anywyere. I think I'd prefer if we raise an exception or something if a `Date` or `EventDateTime` is passed until we better understand the behavior we'd want in that case.

Reply:
I want to talk about this one a little bit. The `HasDuration` module expects attributes called `start` and `end` that behave like `Google::Apis::CalendarV3::EventDateTime`. So that early return is there so that the share examples can stuff the including object with an event_date_time object rather than just a date. I think i need to talk to you about this in person for advice, cause I'm not quite sure about the best way to proceed.

Reply:
Oh, sorry, I actually misread this method because the naming confused me.

This method doesn't actually ever take a `Date`, it only ever takes a `DateTime`, `Time`, or `EventDateTime`, so I think we should change the variable name as well to `datetime`.